### ISSUE TITLE
How to send my own UUID?

### ISSUE THREAD

**Issue by raghsahu** [MrHertal/react-native-twilio-phone#112]
Why you are using random number for uuid. If I have my own UUID, then no option avail in initialize for pass uuid.

**Comment by developersofmen** [comment]
did you get any success?

**Comment by raghsahu** [comment]
Yes, I have used custom class for RNTwillioCall & pass own UUID.

   RNTwilioPhoneCall.initialize(
        callKeepOptions,
        token,
        options,
        userUUID
      );
    }
